docs/docs/pages/examples/multimodal/multimodal-images.mdx

```diff
@@ -0,0 +1,116 @@
+import { RefLink } from "../../../components/RefLink";
+
+# Multimodal Image Analysis [Use Case]
+
+This page shows how to write a Scenario test where the user provides **text _and_ an image** in the same message and the agent must respond appropriately.
+
+## Overview
+
+The test covers three situations:
+
+1. User sends text _and_ an image (agent should describe the image and answer the question).
+2. User sends **only** an image (agent should produce a useful description without textual hints).
+3. User asks a **complex multi-part** question about the image (agent should address every sub-question).
+
+Success is judged automatically by a <RefLink link={{ python: "judge_agent.html#scenario.judge_agent.JudgeAgent", typescript: "functions/judgeAgent.html" }} code={{ python: "JudgeAgent", typescript: "judgeAgent" }} /> with explicit criteria.
+
+## Code Walk-through (TypeScript)
+
+:::code-group
+
+```typescript [typescript]
+import * as fs from "fs";
+import * as path from "path";
+import { openai } from "@ai-sdk/openai";
+import scenario, { type AgentAdapter, AgentRole } from "@langwatch/scenario";
+import { generateText } from "ai";
+
+// 1️⃣ Build an image-capable agent
+const imageAgent: AgentAdapter = {
+  role: AgentRole.AGENT,
+  async call(input) {
+    const response = await generateText({
+      model: openai("gpt-4o"),
+      messages: [
+        {
+          role: "system",
+          content: `You are a helpful assistant that can describe images.`,
+        },
+        ...input.messages,
+      ],
+    });
+    return response.text;
+  },
+};
+
+// 2️⃣ Utility to embed the fixture image as a data-URL
+function getDataURL(): string {
+  const imagePath = path.join(__dirname, "fixtures", "scenario.webp");
+  const buffer = fs.readFileSync(imagePath);
+  return `data:image/webp;base64,${buffer.toString("base64")}`;
+}
```

Comment:
maybe pass the file as argument here, like in the fixtures docs? it's just a bit nicer to later read on the script

Reply:
![image](https://github.com/user-attachments/assets/9165b3aa-4632-4883-b5c7-5ed41eb1a681)